### ISSUE TITLE
Frame limiting

### ISSUE THREAD

**Issue by derekdreery** [amethyst/space-menace#21]
On my machine space marines uses about 50% of a processor. This seems high to me, although maybe it isn't an issue. Amethyst supports frame limiting as a way to reduce unnecessary cpu usage.

**Comment by krankur** [comment]
Yes, this is a know issue and seems to be originating from the Rayon crate. For more details check [here](https://community.amethyst.rs/t/high-cpu-usage-issue/866/6).
